app/views/shared/_search.html.erb

```diff
@@ -0,0 +1,128 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= govuk_error_summary(@search_form) %>
+    <% if title %>
+      <h1 class="govuk-heading-xl"><%= t("#{translation_prefix}.page_title") %></h1>
+    <% end %>
+    <div class="search-panel govuk-!-margin-bottom-6">
+      <%= form_with(url: form_url, method: :get, model: @search_form) do |f| %>
+        <div class="govuk-form-group govuk-!-width-two-thirds" role="search">
+          <%= f.govuk_text_field :query, label: { size: 'm', text: t("#{translation_prefix}.query.label") }, hint: { text: t("#{translation_prefix}.query.hint") } %>
+        </div>
+        <% if choose_application_type && FeatureFlags.nsm_insights.enabled? %>
+          <%= f.govuk_collection_radio_buttons :application_type,
+            @search_form.application_types,
+            :value,
+            :label,
+            inline: true,
+            legend: { text: t("#{translation_prefix}.application_type.label") }
+          %>
+        <% end %>
+        <h2 class="govuk-heading-m"><%= t('.filter_results') %></h2>
+        <div>
+          <div class="govuk-date-input__item">
+            <div class="govuk-form-group">
+              <%= f.govuk_date_field :submitted_from, legend: { size: 's', text: t('.submitted_from') } %>
+            </div>
+          </div>
+          <div class="govuk-date-input__item">
+            <div class="govuk-form-group">
+              <%= f.govuk_date_field :submitted_to, legend: { size: 's', text: t('.submitted_to') } %>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="govuk-date-input__item">
+            <div class="govuk-form-group">
+              <%= f.govuk_date_field :updated_from, legend: { size: 's', text: t('.updated_from', class: 'xs-legend') } %>
+            </div>
+          </div>
+          <div class="govuk-date-input__item">
+            <div class="govuk-form-group">
+              <%= f.govuk_date_field :updated_to, legend: { size: 's', text: t('.updated_to') } %>
+            </div>
+          </div>
+        </div>
+        <div>
+          <div class="govuk-date-input__item">
+            <div class="govuk-form-group">
+              <%= f.govuk_collection_select :status_with_assignment,
+                                            @search_form.statuses,
```

Comment:
Slight issue here - in prior authority, sent back applications have status `sent_back`, in NSM they have status `further_info`. As it stands, this dropdown only has `sent_back`. It _might_ be we should update it so that we use `sent_back` for NSM too, otherwise as it stands we don't have a single status that we can search on to find both NSM _and_ PA sent back submissions.

Reply:
makes sense - if anything, further_info could apply to both but doesn't necessarily whereas sent_back works for both. For now I'll separate that out into a different ticket in case we want to discuss it as a team properly and decide on what's most appropriate